code/modules/vehicles/mecha/combat/phazon.dm

```diff
@@ -3,21 +3,23 @@
 	name = "\improper Phazon"
 	icon_state = "phazon"
 	base_icon_state = "phazon"
+	allow_diagonal_movement = TRUE
```

Comment:
kinda a big buff, but at the same time not, leaving comment here to remind myself to watch for this